README.md

```diff
@@ -5,8 +5,11 @@ Simple command line tool for fetching the [Open Container Initiative image forma
 ## Usage
 
 ```
-$ go get github.com/containers/oci-fetch
+$ go get github.com/containers/oci-fetch/oci-fetch
```

Comment:
Don't disagree.. just making it work.  Can move it add a cmd prefix directory for oci-fetch with another PR if you like.